modules/stochastic_tools/src/controls/SamplerReceiver.C

```diff
@@ -9,110 +9,19 @@
 
 // MOOSE includes
 #include "SamplerReceiver.h"
-#include "Function.h"
 
-registerMooseObject("StochasticToolsApp", SamplerReceiver);
+registerMooseObjectDeprecated("StochasticToolsApp", SamplerReceiver, "01/01/2020 24:00");
```

Comment:
Use `registerMooseObjectReplaced` instead.

Reply:
Would this work since the app changed from StochasticToolsApp to mooseApp?